index.html

```diff
@@ -28,6 +29,69 @@ <h3>I'm a Software Developer.</h3>
                 <li><a href="https://twitter.com/Luffytaro_san22" target="_blank" rel="noopener"><i class="fa-brands fa-twitter"></i></a></li>
             </ul>
         </section>
+        <section id="works">
+            <h2>My Recent Works</h2>
+            <img src="./Normal Button/Tertiary/Icons/Img Placeholder.png" alt="Works-placeholder">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selections of privately personalized reads; no accounts or sign-ups required. Take your time and see the cool works I've done.</p>
+            <ul class="languages">
+                <li>CSS</li>
+                <li>HTML</li>
+                <li>Javascript</li>
+            </ul>
+            <a href=""><button type="button">See project</button></a>
+            <div class="works-container">
+                <div class="works">
+                    <h2>Calculator</h2>
+                    <p>A simple calculator that can do the basics operations.</p>
```

Comment:
- [ ] Base on the figma design the project  cards consist of six cards please kindly update the project card to complete  as exact the figma template design so that you website will  be  more meaningful